### ISSUE TITLE
Type error when tweened object is a Three.Vector3

### ISSUE THREAD

**Issue by MasatoMakino** [tweenjs/tween.js#554]
## Expected Behavior

new tween instance without a type error.

## Current Behavior

error TS2345 is It happens, ``new TWEEN.Tween(cube.position)``

## Possible Solution

TWEEN.Tween(cube.position as any), type errors are not occur.
It is not happen in tween.js 18.5.0 + @types/tween.js 18.5.1

## Steps to Reproduce

Reproduction Repository
https://github.com/MasatoMakino/behavior-of-new-Tween.tween

1. clone repository
2. ``npm run tsc``
3. error TS2345

error log
```
node_modules/three/src/core/BufferAttribute.d.ts:164:10 - error TS2304: Cannot find name 'Iterable'.

164     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:174:10 - error TS2304: Cannot find name 'Iterable'.

174     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:184:10 - error TS2304: Cannot find name 'Iterable'.

184     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:194:10 - error TS2304: Cannot find name 'Iterable'.

194     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:204:10 - error TS2304: Cannot find name 'Iterable'.

204     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:214:10 - error TS2304: Cannot find name 'Iterable'.

214     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:224:10 - error TS2304: Cannot find name 'Iterable'.

224     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:234:10 - error TS2304: Cannot find name 'Iterable'.

234     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:244:10 - error TS2304: Cannot find name 'Iterable'.

244     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

src/TestNewTween.ts:7:31 - error TS2345: Argument of type 'Vector3' is not assignable to parameter of type 'Record<string, unknown>'.
  Index signature is missing in type 'Vector3'.

7 const tween = new TWEEN.Tween(cube.position).to({x: 100, y: 100, z: 100}, 10000); // NG
                                ~~~~~~~~~~~~~
```

## Context (Environment)

Environment
```package.json
"@tweenjs/tween.js": "^18.6.0",
"three": "^0.118.1",
"typescript": "^3.9.5"
```

I was trying to animate the Mesh object in three.js with tween.js.

I referred to the code in the tween.js user guide, but I get a type error.
[tween.js user guide](https://github.com/tweenjs/tween.js/blob/master/docs/user_guide.md#tweenjs-user-guide)

I couldn't determine if the problem was due to my typescript configuration or the type definition file in 18.6.0. If anyone has any information, please let me know.

Thank you.

**Comment by mikebolt** [comment]
This is a problem with the typings. Try to use `as any` until this is fixed.

**Comment by trusktr** [comment]
Thanks for reporting! Yep, the type support is new. Let's see how to
improve it...

#!/JoePea

**Comment by MasatoMakino** [comment]
I created a minimal code in TypeScript Playground that reproduces this problem.
[TypeScript Playground + v3.5.1](https://www.typescriptlang.org/play?ts=3.5.1#code/CYUwxgNghgTiAEAXAngBwQVQHYGssHsB3LABRn1QGd4BeeAJXHxmAB5LEYBLLAcwBp4AV1wFiAPgDcAKEhRK1ACqEQILK0XwQAD0Rrg1bHiKlyVcfADe0+Lfhh8WDjCFhEzABQB9fACMAVuCIAFzwigCUlgC+MlGy0ArwAGpBzADM1nb2js6u7jAe2gD8wVhCALa+IDD8yCVlldX8AF71FVUwkXFxsjmISCpqAAwAjLTwWCCEYYNYHtHhMg5O-YizQwBM45PTyqpzO8mpMGke4YvSQA)

This problem was caused by a breaking change in TypeScript 3.5.
[Breaking Changes : { [k: string]: unknown } is no longer a wildcard assignment target](https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#-k-string-unknown--is-no-longer-a-wildcard-assignment-target)
This problem does not exist in TypeScript 3.3.
[TypeScript Playground + v3.3.3](https://www.typescriptlang.org/play?ts=3.3.3#code/CYUwxgNghgTiAEAXAngBwQVQHYGssHsB3LABRn1QGd4BeeAJXHxmAB5LEYBLLAcwBp4AV1wFiAPgDcAKEhRK1ACqEQILK0XwQAD0Rrg1bHiKlyVcfADe0+Lfhh8WDjCFhEzABQB9fACMAVuCIAFzwigCUlgC+MlGy0ArwAGpBzADM1nb2js6u7jAe2gD8wVhCALa+IDD8yCVlldX8AF71FVUwkXFxsjmISCpqAAwAjLTwWCCEYYNYHtHhMg5O-YizQwBM45PTyqpzO8mpMGke4YvSQA)

The documentation of  breaking changes included a description of the solution.

> Depending on the intended behavior of { [s: string]: unknown }, several alternatives are available:
> 
> + { [s: string]: any }
> + { [s: string]: {} }
> + object
> + unknown
> + any
>
> We recommend sketching out your desired use cases and seeing which one is the best option for your particular use case.
